src/ui/pokedex-page-ui-handler.ts

```diff
@@ -127,8 +126,7 @@ interface SpeciesDetails {
   shiny?: boolean,
   formIndex?: number
   female?: boolean,
-  variant?: number,
-  forSeen?: boolean, // default = false
+  variant?: number
```

Comment:
```suggestion
  variant?: number,
```
You should leave the trailing `,` in situations like this (it reduces the changes in the diff view).